milatools/datasets/split.py

```diff
@@ -0,0 +1,64 @@
+from dataclasses import dataclass
+
+import numpy
+
+
+@dataclass
+class Split:
+    """Returns 3 Splits of the main data set. The splits are indices of the samples inside the data set"""
+
+    train: numpy.array
```

Comment:
was this meant to be torch.Tensor? these are loaded from the torchvision location anyway

Reply:
no, those are indices not the actual sample, those a simple 1-dim array